GTMAppAuth/Sources/KeychainStore/KeychainStore.swift

```diff
@@ -101,10 +102,20 @@ public final class KeychainStore: NSObject, AuthSessionStore {
   @objc(saveAuthSession:error:)
   public func save(authSession: AuthSession) throws {
```

Comment:
Why not call `self.save(authSession: authSession, withItemName: self.itemName)`? Then you don't have to do the accessibility stuff twice?

Reply:
Good idea. Done. :)